be/src/runtime/runtime_state.h

```diff
@@ -221,6 +221,8 @@ class RuntimeState {
 
     int64_t num_bytes_load_total() { return _num_bytes_load_total.load(); }
 
+    int64_t num_bytes_load_scanner_total() { return _num_bytes_load_scanner_total.load(); }
```

Comment:
What is diff between this and `_num_bytes_load_total` ?

Reply:
load_scanner is the decompressed data size.
"Users may wonder why loading such ‘small’ files takes hours. We need to explain the data size to them"